charts/ai-software-templates/chatbot/README.md

```diff
@@ -30,10 +30,13 @@ The helm chart can be directly installed from the OpenShift Dev Console. Check [
 
 ### Install using Helm
 
-In order to install AI Software Template Helm chart using helm directly, you can run:
+In order to install Chatbot AI Sample Helm chart using helm directly, you can run:
 
 ```
-helm upgrade --install <release-name> --namespace <release-namespace> .
+export APP_NAMESPACE='the-namespace-your-release-will-be-installed'
+export RELEASE_NAME='the-name-of-your-release'
+
+helm upgrade --install $RELEASE_NAME --namespace $APP_NAMESPACE .
```

Comment:
Why export these env vars instead of just using the strings inline in the helm chart command?

Reply:
FYI I've reverted this commit. Only because the rest of the changes are more urgent. We can continue the discussion and see what is best!